general-concepts/package-maintainers/text.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0"?>
+<guide self="general-concepts/package-maintainers/">
+<chapter>
+<title>Package Maintainers</title>
+<body>
+
+<p>
+Package maintainers are responsible for ebuild maintenance tasks <d/>
+bumping ebuilds to new upstream releases, updating them as necessary
+with regards to policies, replying to bugs, etc. The maintainers
+of a package are listed in the package's <uri
+link="::ebuild-writing/misc-files/metadata/">metadata.xml</uri> file.
+When assigning bugs, the first maintainer listed becomes the bug's
+assignee and the remaining maintainers are added to CC, unless stated
+otherwise in the metadata.
+</p>
+
+<section>
+<title>Maintainer authority</title>
+<body>
+
+<p>
+Package maintainers have authority over their maintained packages.
+Unless specified otherwise, you should request maintainers' approval
+before committing to their packages. Try filing a bug, catching them
+on IRC or sending a mail first. The exceptions to this rule are trivial
+changes implied by your own actions, e.g. dependency updates as a result
+of package moves.
+</p>
+
+<p>
+In case of maintainer inactivity, it is a good idea to consult other
+developers on how to handle the situation, especially if it is
+a critical issue that needs to be handled ASAP. A customary tool
+of maintainer timeout may be used in this case. This timeout can be
+invoked if none of the package's maintainers reply to a patch within
+2<d/>4 weeks of it being attached or linked to a correctly assigned bug,
+or being sent to the maintainer.
```

Comment:
Just want to clarify that this change removes the IRC out of the equation. The previous version allows you to establish inactivity from the point when the developer files a bug, sends an email, or messages the maintainer on IRC. This wording requires the developer to come up with a patch before they can establish inactivity. In other words, I can't anymore say "hey, this is broken. the bug has been open for X amount of time. it's blocking my work. can you please fix it?", wait for 2 weeks, and fix it myself. Yes, if I fix it myself, inevitably I'll have to come up with that patch, but not until the timeout happens according to the previous version of the paragraph.

Are we on the same page with the implications of this change? Other than that, everything looks fine.

Reply:
Yes, that's the idea. I want to avoid basically the situation when someone reports a hard issue, waits a month, then attaches a really bad patch and commits it without giving the developer time to review. I mean, I've seen that happen.

I don't mean we should let developers block something forever waiting for the perfect patch. But we shouldn't conflate inactivity with inability to find a solution. It's one thing when developer doesn't review suggested fixes. It's another when he rejects them (and in that case you ought to appeal rather than claim maintainer timeout).